src/main.rs

```diff
@@ -62,10 +63,6 @@ async fn main() {
         ));
     }
 
-    if let Err(e) = config.validate_set_up().await {
-        panic!("Could not validate the supplied configurations: {e}")
-    }
-
     // Using unwrap directly as the query has been ran in the set-up validation
     let wallet = build_wallet(config.wallet_input().unwrap()).unwrap();
```

Comment:
This and following unwrap on config fields are no longer safe to do before configurations only gets validated after GraphcastAgentConfig is used for initializing GraphcastAgent. The users would lose the helpful logs from validating the setup. 

To retain the helpful error messages, maybe move initialization of GraphcastAgent (thus config validation up), or add setup validation to radio specific configs.

Reply:
great point, I'll probably move the GraphcastAgent init up